src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/src/QuickGrid.razor

```diff
@@ -66,7 +66,15 @@
     private void RenderRow(RenderTreeBuilder __builder, int rowIndex, TGridItem item)
     {
         var rowClass = RowClass?.Invoke(item);
-        <tr @key="@(ItemKey(item))" aria-rowindex="@rowIndex" class="@rowClass">
+        var combinedClass = OnRowClick.HasDelegate
+            ? string.IsNullOrEmpty(rowClass) ? "row-clickable" : $"row-clickable {rowClass}"
+            : rowClass;
+        var rowClick = OnRowClick.HasDelegate ? EventCallback.Factory.Create<MouseEventArgs>(this, () => OnRowClick.InvokeAsync(item)) : default;
+
+        <tr @key="@(ItemKey(item))"
```

Comment:
How does the row click behave when it comes to propagation? Will a click on the <tr> element bubble up to its ancestor DOM element? Do we have to support `stopPropagation`?